templates/gmsim/20.4.1.4/root_defaults.yaml

```diff
@@ -15,6 +15,4 @@ emod3d:
    emod3d_version: 3.0.8
 flo: 0.25
 dt: 0.02
-global_root: /nesi/project/nesi00213
-v_1d_mod: /nesi/project/nesi00213/VelocityModel/Mod-1D/Cant1D_v3-midQ_OneRay.1d
```

Comment:
Where is this set now?

Reply:
global_root is never used. v_1d_mod is currently hard coded into install fault...

Reply:
Unless overridden by the user in the pre processing stages (ie Sarahs latest feature request)

Reply:
Ah my reply didn't come through.

"Sweet. Global root is "useless".

v1d is what we use for HF right? Though I guess it is perturbatable so it's probably okay for now. Put a bug in Jira do you reckon that's the best?"

Reply:
Yeah, we could add an argument for it to install_cybershake, but that feels like a problem for when we have a different non-perturbation related base 1d vm. You can use --common-source-parameter <1dvm> with pre-pro to use a different one for all too

Reply:
I think root params is probably the best place for it. But it's a bug that it doesn't propogate through.

Reply:
I can add v_1d_mod back with just the file name to the template root params, then the install adds on the platform VM dir path to the start of it